### PR TITLE
0.6.2

### PR DESCRIPTION
**Pull request by dsherret**
Bumping version to 0.6.2.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "crossterm_winapi"
-version = "0.6.1"
+version = "0.6.2"
```
Comment (dsherret) [inline Cargo.toml:3]: Only one PR since the last release that removed some printlns: https://github.com/crossterm-rs/crossterm-winapi/commits/master

Comment (TimonPost) [reply Cargo.toml:3]: Thanks! Forgot this update :)

**Review by TimonPost — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (dsherret) [reply Cargo.toml:3]: @TimonPost sorry to bother, but anything I can do to help get out a release of this and https://github.com/crossterm-rs/crossterm/pull/498 to crates.io?

Comment (TimonPost) [reply Cargo.toml:3]: The only thing to do is view crates io now ;)

**Pull request merged**